src/database/queries/modules.js

```diff
@@ -0,0 +1,75 @@
+const Modules = require('@database/models/index').Module
+const { Op } = require('sequelize')
+
+module.exports = class UserRoleModulesData {
+	static async createModules(data) {
+		try {
+			return await Modules.create(data, { returning: true })
+		} catch (error) {
+			throw error
+		}
+	}
+
+	static async findModulesById(id) {
+		try {
+			return await Modules.findByPk(id)
+		} catch (error) {
+			throw error
+		}
+	}
+
+	static async findAllModules(page, limit, search) {
```

Comment:
@sumanvpacewisdom where condition can be passed from the service, so this function will be generic